mm2src/coins/z_coin.rs

```diff
@@ -1127,7 +1129,7 @@ impl<'a> ZCoinBuilder<'a> {
             Some(file_path) => PathBuf::from(file_path),
         };
 
-        #[cfg(test)]
+        #[cfg(all(test, not(target_arch = "wasm32")))]
```

Comment:
the whole function is already tagged with `not(wasm32)`, we don't need this extra gating.